types/rest/rest.go

```diff
@@ -229,21 +230,31 @@ func ParseQueryHeightOrReturnBadRequest(w http.ResponseWriter, cliCtx context.CL
 
 // PostProcessResponseBare post processes a body similar to PostProcessResponse
 // except it does not wrap the body and inject the height.
-func PostProcessResponseBare(w http.ResponseWriter, cliCtx context.CLIContext, body interface{}) {
+func PostProcessResponseBare(w http.ResponseWriter, ctx context.CLIContext, body interface{}) {
 	var (
 		resp []byte
 		err  error
 	)
 
+	// TODO: Remove once client-side Protobuf migration has been completed.
```

Comment:
Please open an issue for this TODO item so that we don't forget about it :+1:

Reply:
...or even better add it to the checklist in the issue that you mentioned below https://github.com/cosmos/cosmos-sdk/issues/5864

Reply:
There already is one and it's stated right below...

Reply:
yes and you're right, nevermind! :+1: